From 08ce89d3088dc0a6dd9d3b7455222aa35894c1d5 Mon Sep 17 00:00:00 2001
From: Nikita Popov <npopov@redhat.com>
Date: Thu, 19 Jan 2023 14:28:58 +0100
Subject: [PATCH] [Flang] Explicitly include cstdint (NFC)

This header uses std::int8_t, but does not include cstdint.

This fixes the build against libstc++ 13, where some indirect
header includes have been removed.
---
 flang/include/flang/Parser/characters.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/flang/include/flang/Parser/characters.h b/flang/include/flang/Parser/characters.h
index 1a7d395be2bf7..239be8d4bf4aa 100644
--- a/flang/include/flang/Parser/characters.h
+++ b/flang/include/flang/Parser/characters.h
@@ -14,6 +14,7 @@
 // also to accomodate Fortran tokenization.
 
 #include <cstddef>
+#include <cstdint>
 #include <optional>
 #include <string>
 
